mne/annotations.py

```diff
@@ -608,4 +617,20 @@ def events_from_annotations(raw, event_id=None, regexp=None,
     logger.info('Used Annotations descriptions: %s' %
                 (list(event_id_.keys()),))
 
+    if len(dropped) > 0:
```

Comment:
even if this is only used once, we could make a function that handles dropped events